gradle/docs.gradle

```diff
@@ -119,9 +104,12 @@ asciidoctor {
 		jvmArgs '--add-opens', 'java.base/sun.nio.ch=ALL-UNNAMED', '--add-opens', 'java.base/java.io=ALL-UNNAMED'
 	}
 	baseDirFollowsSourceFile()
-	configurations 'asciidoctorExt'
-	sourceDir "$buildDir/asciidoc"
+	configurations 'asciidoctorExtensions'
+	sourceDir "src/reference/asciidoc/"
 	inputs.dir(sourceDir)
+	outputOptions {
+		backends "spring-html"
```

Comment:
Just curious if you have build the doc for the project and checked if it looks as expected in both HTML and PDF.
Plus HTML-single.

Reply:
I did, but, on checking again after you asked, I found two places that need work. I'll look into them. Thanks.

Reply:
It's all fixed now. Thanks, Artem.